grpc-sys/build.rs

```diff
@@ -67,6 +67,8 @@ mod imp {
         prepare_grpc();
 
         let dst = CMakeConfig::new("grpc")
+            .cflag("-w")
```

Comment:
I don't think it's a good idea. Can you raise an issue to the upstream project that fails to compile?